docs/reference/manifest/override.md

```diff
@@ -184,3 +185,56 @@ An `<Override>` element expresses a conditional and can be read as an "If ... th
 - [Office versions and requirement sets](../../develop/office-versions-and-requirement-sets.md)
 - [Set the Requirements element in the manifest](../../develop/specify-office-hosts-and-api-requirements.md#set-the-requirements-element-in-the-manifest)
 - [Keyboard shortcuts](../../design/keyboard-shortcuts.md)
+
+## Override element of type RuntimeOverride (preview)
+
+> [!IMPORTANT]
+> This feature is only supported for [preview](../../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) in Outlook on the web and on Windows with a Microsoft 365 subscription. For more details, see [Configure your Outlook add-in for event-based activation](../../outlook/autolaunch.md).
+>
+> Because preview features are subject to change without notice, they shouldn't be used in production add-ins.
+
+An `<Override>` element expresses a conditional and can be read as an "If ... then ..." statement. If the `<Override>` element is of type **RuntimeOverride**, then the `type` attribute is the condition, and the `resid` attribute is the consequent. For example, the following is read "If the type is 'javascript', then the `resid` is 'JSRuntime.Url'." Outlook Desktop requires this element for [LaunchEvent extension point](../../reference/manifest/extensionpoint.md#launchevent-preview) handlers.
+
+```xml
+<Runtime resid="WebViewRuntime.Url">
+  <Override type="javascript" resid="JSRuntime.Url"/>
+</Runtime>
+```
+
+**Add-in type:** Mail
+
+### Syntax
+
+```XML
+<Override type="javascript" resid="JSRuntime.Url"/>
+```
+
+### Contained in
+
+- [Runtime](runtime.md)
+
+### Attributes
+
+|Attribute|Type|Required|Description|
+|:-----|:-----|:-----|:-----|
+|**type**|string|Yes|Specifies the language for this override. At present, `"javascript"` is the only supported option.|
+|**resid**|string|Yes|Specifies the URL location of the JavaScript file that should override the URL location of the default HTML defined in the parent `Runtime` element's `resid`. The `resid` can be no more than 32 characters and must match an `id` attribute of a `Url` element in the `Resources` element.|
```

Comment:
Can we make the elements link to their pages when possible? Like `Runtime` should link to the Runtime element page. Makes it easier to follow the dots when working on the manifest.